### DISCUSSION TITLE
モバイル学習コンテンツにStackNavigatorの仕様を記載しても良いのでは

### DISCUSSION THREAD

**Discussion by nagano-ryo** [ws-4020/mobile-app-crib-notes#249] [Ideas]
学習コンテンツFBより

https://ws-4020.github.io/mobile-app-crib-notes/react-native/learn/basic-concepts/react-navigation-basics

特に初期表示画面についての説明があると良い
・initialRouteNameが定義されていない場合　→　最初に定義した画面
・initialRouteNameが定義されている場合　→　initialRouteNameで定義した画面

なお、initialRouteNameが定義されている例は以下にあり。
https://ws-4020.github.io/mobile-app-crib-notes/react-native/learn/todo-app/screens/welcome

**Comment by nagano-ryo** [comment]
pull request
https://github.com/ws-4020/mobile-app-crib-notes/pull/267

**Comment by creature-water-valley** [comment]
対応済みなのでクローズ
